src/views/documents/utils/QueryItems.vue

```diff
@@ -222,7 +222,7 @@ export default {
     },
 
     documentType() {
-      return this.$route.name.slice(0, -1);
+      return this.$route.name.split('-')[0].slice(0, -1);
```

Comment:
a little bit hacky 🙂

Reply:
Yes :/

I've added a comment to explain the hack :p